### PR TITLE
[JENKINS-59648] UX improvements

### PR DESCRIPTION
**Pull request by aheritier**
Ref: https://issues.jenkins-ci.org/browse/JENKINS-59648

Few screenshots

Screenshot 2019-10-04 00 09 30

Screenshot 2019-10-04 00 09 11
Screenshot 2019-10-04 00 11 13
Screenshot 2019-10-04 00 08 53
Screenshot 2019-10-04 00 08 34

### CODE REVIEW

File: src/main/resources/lib/advisor/validateOnLoad.jelly [hunk 1]
```diff
@@ -18,21 +18,22 @@
...
-            <p>${validationStatus}</p>
+            <strong>Terms and Conditions not accepted.</strong>
+            <p>To use Jenkins Health Advisor, please agree to <a href='https://www.cloudbees.com/jenkins/cloudbees-jenkins-advisor/terms-of-service'>our Terms and Conditions</a>.</p>
           </div>
         </j:if>
-        <j:if test="${validationStatus.contains('Terms of Service not accepted')}">
+        <j:if test="${validationStatus.contains('xception')}">
           <div class="alert alert-warning" role="alert">
-            <h4 class="alert-heading">${validationStatus}</h4>
+            <strong>Connection failure to Jenkins Health Advisor by CloudBees.</strong>
```
Comment (kwhetstone) [inline src/main/resources/lib/advisor/validateOnLoad.jelly]: Aside: what should we say to do when a user hits these issues?  Some would probably be easy to solve on their own, but should we provide a link out to documentation or say something like "Visit the CloudBees support site for information"

Comment (aheritier) [reply src/main/resources/lib/advisor/validateOnLoad.jelly]: You're right @kwhetstone Let's add a link to the troubleshooting KB https://support.cloudbees.com/hc/en-us/articles/115001213031
We are working in parallel to improve the documentation thus we might improve more all of this in the future

Comment (aheritier) [inline src/main/resources/lib/advisor/validateOnLoad.jelly]:
```suggestion
            <strong>Connection failure to Jenkins Health Advisor by CloudBees.</strong>
            <p>Please check our <a href="https://support.cloudbees.com/hc/en-us/articles/115001213031" target="_blank">Troubleshooting Guide</a> to solve the issue.</p>
```

**Comment by aheritier** [comment]
Based on your feedbacks @kwhetstone I did some improvements in the warning box to add the link to the troubleshooting guide and to give the e-mail used to receive more help.

Screenshot 2019-10-04 09 09 04

**Pull request merged**